modules/ROOT/pages/clustering/servers.adoc

```diff
@@ -300,6 +476,7 @@ neo4j@neo4j> RENAME SERVER '25a7efc7-d063-44b8-bdee-f23357f89f01' TO 'eu-server-
 ----
 
 This only affects the name of the server; the ID of the server remains fixed as `25a7efc7-d063-44b8-bdee-f23357f89f01`.
+Keep in mind that the name of the server must be unique.
```

Comment:
Within existing servers. You can have a server name it 'foo', drop it, and then name a different server 'foo';